scripts/build-l1-image

```diff
@@ -7,6 +7,11 @@
 #
 # The default rpc port is 8545.
 #
+# Part of the process of creating the Docker image is to deploy the L1 contracts required by the
+# demo. The deployment is not completely deterministic; addresses may change. Therefore, this script
```

Comment:
What kind of non-determinism was observed here? The address only depends on the sender account and transaction nonce.